apps/i18n/common/en_us.json

```diff
@@ -813,9 +815,6 @@
   "minecraftAquaticButton": "Go to Minecraft Education",
   "minecraftAquaticDesc": "Continue learning with Minecraft.",
   "minecraftAquaticTitle": "Minecraft Education",
-  "minecraftMarketingButton": "Start",
-  "minecraftMarketingDesc": "Minecraft is back for the Hour of Code with a brand new activity! Journey through Minecraft with code.",
-  "minecraftMarketingTitle": "Minecraft: Hero’s Journey",
```

Comment:
i'm sure you checked, but just double-checking that these "minecraft_" strings aren't used elsewhere

Reply:
Yep, thanks for the reminder!